tutorials/reviewing-tutorials.md

```diff
@@ -24,3 +25,24 @@ These are the main areas for alignment and consistency:
 4. Materials migrate to Github after reaching initial draft status, and reviews continue there
 
 See [Content Review Sign-Up](https://docs.google.com/spreadsheets/d/1LtLEk0H7gkR34kXJVC3hICUsLwfabbSjTPSlterB-xg/edit) for additional instructions
+
+## Workflow for incorporation into the Github repostory
+After initial review, notebooks should be incorporated into the [NMA Github Repository](https://github.com/NeuromatchAcademy/course-content). This involves both manual and automated quality control.
+1. Once initial review is complete, Content Creators should "hand-off" the tutorial to their Review Day Chief. At this time, the tutorial content should be finished, but the notebook may be missing links to the finalized lecture videos. The hand-off is accomplished by having the Review Day Chief make a copy of the tutorial that they now own.
+2. The Review Day Chief should run their copy of the notebook from top-to-bottom on a fresh kernel (`Runtime -> Restart and run all`) and check that it succeeds without error.
+3. The Reivew Day Chief should upload their copy to Github:
+  - "File" -> "Save a copy in Github"
+  - Repository: `NeuromatchAcademy/course-content`
+  - Branch: `WxDy`
+    - Replace x and y with the appropriate week and day (e.g. W2D3)
+  - File path: `tutorials/WxDy-SubjectName/WxDy-Tutoriali.ipynb
+    - Replace `x`, `y`, `i`, and `$SubjectName`
+    - **Important**: Make sure the file name is correct; fixing them will be a pain
+4. The Reivew Day Chief should open a PR from the `WxDy` branch to `master`
+  - Assign Reviewers (@mwaskom, @mpbrigham, @patrickmineault)
```

Comment:
I'm going to be pretty busy this Sunday with student portal stuff, we should find other reviewers in case we don't have enough bandwidth. Byron?

Reply:
Will ask in Slack as we get closer.